osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -240,6 +240,7 @@ private bool onOsuLogo()
         private ButtonSystemState state = ButtonSystemState.Initial;
 
         public override bool HandleNonPositionalInput => state != ButtonSystemState.Exit;
+
```

Comment:
why..., please take a look at the project's code style, we normally have properties/functions/etc.. together without a new line if they're a 1 line and related in some way (e.g `OnMouseDown`, `OnMouseUp`)